benchmarks/matrices.py

```diff
@@ -61,30 +61,30 @@ def time_MutableDenseMatrix_getitem(self):
         m = self.M1
         for i in range(m.rows):
             for j in range(m.cols):
-                m[i, j]
+                _ = m[i, j]
```

Comment:
This seems unnecessary and just makes things more verbose.

Reply:
the `_ =` stands for a throwaway value. The purpose is to avoid writing statements that look like they no effect but are supposed to be like that. The `_ =` signifies that the return value is not used.

I'll remove it.

Reply:
These are benchmarks so it is all about timing the execution. So it is fine to call functions, methods, etc. but not use the result (or indicate that we aren't using them).